t/APISIX.pm

```diff
@@ -671,7 +671,6 @@ _EOC_
     $a6_ngx_directives
 
     server {
-        listen 1983 quic reuseport;
```

Comment:
why did you remove this?

Reply:
this is a https upstream server for test. But nginx does not support HTTP/3 with usptream. So this line does not take effect.